### PR TITLE
chore: Add labels for tracks

### PR DESCRIPTION
**Pull request by oskarth**
Partially addresses https://github.com/vacp2p/research/issues/112

Subset of labels defined here https://github.com/vacp2p/research/blob/master/.github/labels.yml

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  description: This issue is blocked by some other work
+  color: e0af74
+- name: critical
+  description: This issue needs critical attention
+  color: B60205
+- name: infra
+  description: Infra, devops, CI and related tasks
+
+# Waku Product labels
+  color: 277196
```
Comment (s1fr0) [inline .github/labels.yml:41]: This color seems to be out of place

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- name: rqt:platforms
+  description: Requirement coming from a platform
+  color: A51BB5
+
+# Tracks primarily within Secure Messaging or Waku Product project
+- name: track:rln
+  description: RLN Track - (Secure Messaging/Applied ZK), relay and applications
```
Comment (s1fr0) [inline .github/labels.yml:48]: For consistency:

```suggestion
  description: RLN Track (Secure Messaging/Applied ZK), relay and applications
```

File: .github/labels.yml (hunk 1 above)
Comment (oskarth) [reply .github/labels.yml:41]: Fixed

File: .github/labels.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  description: Protocol Incentivization track (Secure Messaging), e.g. service credentials
+  color: 0037E3
+- name: track:restricted-run
+  description: Restricted run track (Secure Messaging/Waku Product), e.g. filter, WebRTC
+  color: D91C35
+- name: track:conversational-security
+  description: Conversational security track (Secure Messaging)
+  color: CC6B00
+- name: track:nwaku-productionization
+  description: nwaku-productionization track (Waku Product)
```
[the remark this answers is not in the document]
Comment (oskarth) [reply .github/labels.yml]: fixed, plus same in 3 entries below https://github.com/status-im/nwaku/pull/1046/commits/646ee508a5542bc39aab6c7c3bf497ef760098a5

**Comment by oskarth** [comment]
Will merge, for further tweaks I suggest we update later, otherwise gets a bit hairy with changes in many places

The master list / issue is in https://github.com/vacp2p/research/issues/112

**Pull request merged**